.devcontainer/rhel-dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM --platform=linux/amd64 redhat/ubi8:8.9
+
+# Install gcc, make, etc
+RUN yum -y install gcc gcc-c++ glibc-devel make cmake
```

Comment:
Do we want to lock in specific versions for stuff like this?

Reply:
I don't think it's important for this application, but we'll keep that in mind moving forward